src/Bicep.LangServer/Handlers/BicepHoverHandler.cs

```diff
@@ -70,25 +71,31 @@ public override Task<Hover> Handle(HoverParams request, CancellationToken cancel
                 case OutputSymbol output:
                     return $"```bicep\noutput {output.Name}: {output.Type}\n```";
 
+                case NamespaceSymbol namespaceSymbol:
+                    return $"```bicep\n{namespaceSymbol.Name} namespace\n```";
+
                 case FunctionSymbol function when result.Origin is FunctionCallSyntax functionCall:
                     // it's not possible for a non-function call syntax to resolve to a function symbol
                     // but this simplifies the checks
-                    return GetFunctionMarkdown(function, functionCall, result.Context.Compilation.GetSemanticModel());
+                    return GetFunctionMarkdown(function, functionCall.Arguments, result.Origin, result.Context.Compilation.GetSemanticModel());
+
+                case FunctionSymbol function when result.Origin is InstanceFunctionCallSyntax functionCall:
+                    return GetFunctionMarkdown(function, functionCall.Arguments, result.Origin, result.Context.Compilation.GetSemanticModel());
 
                 default:
                     return null;
             }
         }
 
-        private static string GetFunctionMarkdown(FunctionSymbol function, FunctionCallSyntax functionCall, SemanticModel model)
+        private static string GetFunctionMarkdown(FunctionSymbol function, ImmutableArray<FunctionArgumentSyntax> arguments, SyntaxBase functionCall, SemanticModel model)
```

Comment:
InstanceFunctionCallSyntax also calls this method, updated signature to be able to call it when the case is either a FunctionCallSyntax or InstanceFunctionCallSyntax